stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/ImageAttachmentsGroupView.kt

```diff
@@ -93,6 +94,11 @@ internal class ImageAttachmentsGroupView : ConstraintLayout {
         val imageAttachmentView = createImageAttachmentView()
         addView(imageAttachmentView)
         state = State.OneView(imageAttachmentView)
+
+        val imageWidth = first.originalWidth?.toFloat()
+        val imageHeight = first.originalHeight?.toFloat()
+        val imageAspectRatio = (imageWidth ?: 0f) / (imageHeight ?: 0f)
```

Comment:
Added a `NaN` check, should be fine now if both are `0f`.

Reply:
Alternatively you could coerceAtLeast to 1f for the height

Reply:
I am not sure if this could introduce any edge cases if width returns `!=0f` and height is, I would leave this just to be on the safe side. But also not sure how likely it is for this to happen.